hs_model_program/static/css/model_program.css

```diff
@@ -0,0 +1,56 @@
+
+.mpElement{
+    font-weight: bold;
+    padding: 2px 2px 2px 2px;
+}
+
+a.mpElement::after {
+    display:block;
+    /*content:attr(title);*/
+    font-weight:bold;
+    height:2px;
+    color:transparent;
+    overflow:hidden;
+    visibility:hidden;
+}
+
+div#mp.row {
+    padding: 5px 0px 5px 0px
+}
+
+option:checked{
+    background: #0073ea;
+}
+table{
```

Comment:
Fixed spacing between selectors and '{'.  817e36322f124c70d4a82767b6ff9cad46ef851b